### ISSUE TITLE
All commands result in error: command not found

### ISSUE THREAD

**Issue by kory-smith** [tonka3000/vscode-raycast#20]
### Steps to reproduce

Install the extension on Vscode and run any command (I'm using "Raycast: Start Development mode"). It'll error. My setup:

Version: 1.74.2
Commit: e8a3071ea4344d9d48ef8a4df2c097372b0c5161
Date: 2022-12-20T10:32:21.587Z (1 wk ago)
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.2.0
Sandboxed: Yes

### Relevant logs and/or screenshots

![image](https://user-images.githubusercontent.com/26067990/210303941-21901abe-074c-4581-a485-ea80716086ac.png)

**Comment by tonka3000** [comment]
Hi @kory-smith 

It looks like that VSCode didn't register the commands at all which is strange.

Some questions:

- Do you see the the Raycast Tree view in the explorer tab?
- What happen when you type `npm exec ray develop` into the integrated terminal?
- What happen when you type `npm run dev` into the integrated terminal?
- Can you choose `Raycast` in the drop-down of the output panel (next to terminal panel)

Greetings
Michael

**Comment by whatyoubendoing** [comment]
I ran into the same issue. 

@tonka3000  The transit folder is not created before the extension tries to use it / watch the folder. 
@kory-smith A temporary fix is creating this folder 

```bash
mkdir -p "/Users/$USER/Library/Application Support/Code/User/globalStorage/tonka3000.raycast/transit"
```

**Comment by tonka3000** [comment]
@Ben2HellAndBack Thanks for the hint. I will ship a hotfix for that.

**Comment by tonka3000** [comment]
@kory-smith  @Ben2HellAndBack Fixed via 44a9566133f1d58ea459614c02919d3bd3df9bc6 (0.9.1)
